src/Uno.UWP/Uno.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="MSBuild.Sdk.Extras" ToolsVersion="15.0">
+﻿<Project Sdk="MSBuild.Sdk.Extras" ToolsVersion="15.0">
```

Comment:
You can rollback all the modifications made to this file. Visual Studio makes them, but they are not useful.